### DISCUSSION TITLE
Embed Docsify in React

### DISCUSSION THREAD

**Discussion by junoriosity** [docsifyjs/docsify#1906] [Q&A]
I just came across docsify and must say, I really consider it to be looking extremely good.

Currently, I am working on my React frontend and I am wondering whether it is possible to embed docsify into it?

**Comment by ghost** [comment]
Hi junoriosity. 

> I just came across docsify and must say, I really consider it to be looking extremely good.

great feedback.

> Currently, I am working on my React frontend and I am wondering whether it is possible to embed docsify into it?

So... I thought about the idea you talk about here with web components. Please, see this: [docsify with web-components?](https://github.com/docsifyjs/docsify/discussions/1940).

**Comment by jhildenbiddle** [comment]
Closing in favor of #2135.
